docs/platforms/apple/guides/ios/session-replay/customredact.mdx

```diff
@@ -75,3 +75,28 @@ To hide the username, we need to mask it.
     .sentryReplayUnmask() 
   }
 ```
+
+## Debugging session replay masking
+
+To visualize which elements are being masked, you can enable a masking overlay.
```

Comment:
`m`: I'm assuming this means that all masks (custom + built-in) are applied, and they're applied both in Xcode Preview and in the debug app running on simulator or device, correct? If not, we should point it out